### ISSUE TITLE
Selected inputs go through the debugger

### ISSUE THREAD

**Issue by RIMOPA** [ScratchAddons/ScratchAddons#4224]
### Description

When the debugger is moved and cover a selected input, the input go through the debugger.

### Reproduction steps

1. Enable the debugger addon.
2. Select an input.
3. Move the debugger over the selected input.

### Expected behavior

The input is not visible.

### Actual behavior

The input is visible

### Screenshots

![image](https://user-images.githubusercontent.com/97112832/152722680-a9792822-0494-4afd-9235-e999147098cf.png)
### Environment

- Browser: 97.0.4692.99 (Build oficial) (64 bits) (cohort: Stable)
- Scratch Addons version: 1.24.1
- Operating system: Windows

**Comment by mxmou** [comment]
This can happen without Scratch Addons:
![image](https://user-images.githubusercontent.com/51849865/152745256-961f2a1a-74ea-4984-ae24-77ca23d992b6.png)

We could probably fix it anyway.

**Comment by Norbiros** [comment]
To fix it we need to add only:
`cd
.fieldTextInput {
  z-index: 0;
}
`
But to which addon we need to add it? Some addon for developer, that most people have enabled, but it don't fits to developer's tools...
**We can also create PR to scratch-gui...**

**Comment by WorldLanguages** [comment]
> To fix it we need to add only:
> `.fieldTextInput { z-index: 0; }`

I don't think it would be a good idea to change the z-index of vanilla Scratch elements

> We can also create PR to scratch-gui...

Please no :P
